### PR TITLE
Remove the unused vars from the version-def CI script

### PR DESCRIPTION
**Pull request by NvTimLiu**
To fix: https://github.com/NVIDIA/spark-rapids/issues/11223

The definition of [CUDF_VER=24.08.0-SNAPSHOT](https://github.com/NVIDIA/spark-rapids/blob/branch-24.08/jenkins/version-def.sh#L29) is unreferenced/unused, because we'd changed to depend on the spark-rapids-jni jar instead.

Need to remove unreferenced CUDF_VER=xxx in the CI script: https://github.com/NVIDIA/spark-rapids/blob/branch-24.08/jenkins/version-def.sh#L29

### CODE REVIEW

**Comment by NvTimLiu** [comment]
build

File: jenkins/version-def.sh [hunk 1]
```diff
@@ -27,7 +27,6 @@ done
 IFS=$PRE_IFS
 
 
-CUDF_VER=${CUDF_VER:-"24.08.0-SNAPSHOT"}
```
Comment (jlowe) [inline jenkins/version-def.sh:30]: This variable is echo'd below, need to remove that as well.

**Comment by NvTimLiu** [comment]
build

**Review by pxLi — APPROVED** [review]

**Review by gerashegalov — APPROVED** [review]

**Comment by NvTimLiu** [comment]
CI failed with ConnectionRefusedError

```
[2024-07-23T01:50:04.593Z] >           self.socket.connect((self.java_address, self.java_port))
[2024-07-23T01:50:04.593Z] E           ConnectionRefusedError: [Errno 111] Connection refused
[2024-07-23T01:50:04.593Z] /home/jenkins/agent/workspace/jenkins-rapids_premerge-github-9797-ci-2/.download/spark-3.2.0-bin-hadoop3.2/python/lib/py4j-0.10.9.2-src.zip/py4j/clientserver.py:402: ConnectionRefusedError
```

**Comment by NvTimLiu** [comment]
build

**Review by jlowe — APPROVED** [review]

**Comment by NvTimLiu** [comment]
build

**Comment by NvTimLiu** [comment]
build

**Pull request merged**